src/xcode/ENA/ENA/Source/Scenes/Events/Store/Model/TraceLocation.swift

```diff
@@ -104,9 +129,19 @@ struct TraceLocation {
 		return Date() < endDate
 	}
 	
+	var guidHash: Data? {
+		let encoder = JSONEncoder()
+		do {
+			let guidData = try encoder.encode(guid)
+			return guidData.sha256()
+		} catch {
+			Log.error("traceLocationGUID Encoding error", log: .checkin, error: error)
+			return nil
+		}
+	}
```

Comment:
Sorry, missed that one yesterday, there is no need to use the JSON encoder:

```suggestion
	var guidHash: Data {
	    return guid.data(using: .utf8).sha256()
	}
```
Please review the indentation. Tabs suck!

Reply:
done, thanks!